src/docs/output-targets/dist.md

```diff
@@ -32,38 +32,37 @@ Luckily, both builds can be generated at the same time, and shipped in the same
+| `dir`                                     | The `dir` config specifies the public distribution directory. This directory is commonly the `dist` directory found within [npm packages](https://docs.npmjs.com/getting-started/packages). This directory is built and rebuilt directly from the source files. Additionally, since this is a build target, all files will be deleted and rebuilt after each build, so it's best to always copy source files into this directory. It's recommended that this directory not be committed to a repository.                                                                                                                                                                                                                           | `dist`  |
+| `empty`                                   | By default, before each build the `dir` directory will be emptied of all files. To prevent this directory from being emptied, change this value to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | `true`  |
+| `transformAliasedImportPathsInCollection` | This option will allow [path aliases](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) defined in a project's `tsconfig.json` to be transformed into relative paths in the output code for the implicit `dist-collection` output target (i.e. something like `import * as utils from '@my-utils-path';` will become `import * as utils from '../path/to/utils';`). This does not affect imports for external packages. Currently, aliased paths are not transformed in the emitted code after running through the TS compiler.<br><br>If using the `dist-collection` output target directly, the same result can be achieved using the `transformAliasedImportPaths` flag on the target's config. | `false` |
```

Comment:
No action required - this table isn't properly formatted, but wasn't to begin with.  We could reformat it like so:
```suggestion
| Property                                  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Default |
|-------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
| `dir`                                     | The `dir` config specifies the public distribution directory. This directory is commonly the `dist` directory found within [npm packages](https://docs.npmjs.com/getting-started/packages). This directory is built and rebuilt directly from the source files. Additionally, since this is a build target, all files will be deleted and rebuilt after each build, so it's best to always copy source files into this directory. It's recommended that this directory not be committed to a repository.                                                                                                                                                                                                                           | `dist`  |
| `empty`                                   | By default, before each build the `dir` directory will be emptied of all files. To prevent this directory from being emptied, change this value to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | `true`  |
| `transformAliasedImportPathsInCollection` | This option will allow [path aliases](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) defined in a project's `tsconfig.json` to be transformed into relative paths in the output code for the implicit `dist-collection` output target (i.e. something like `import * as utils from '@my-utils-path';` will become `import * as utils from '../path/to/utils';`). This does not affect imports for external packages. Currently, aliased paths are not transformed in the emitted code after running through the TS compiler.<br><br>If using the `dist-collection` output target directly, the same result can be achieved using the `transformAliasedImportPaths` flag on the target's config. | `false` |
```
but like I said, it wasn't like this beforehand, and this comment contradicts my advice in the next comment - take a look at that first and we can go from there